scripts/install-chef-suse.sh

```diff
@@ -522,7 +522,7 @@ for arch in $product_arches; do
         if ! is_ses; then
```

Comment:
Not visible here, but on line 518, you probably need to change SLES12-SP1 to SLES12-SP2 (unless SP2 still has the bug where it pretends to be SP1...)